### DISCUSSION TITLE
Tooltip: allow tooltip activation for both 'hover' and 'focus'

### DISCUSSION THREAD

**Discussion by barbGit2** [primefaces/community#1409] [PrimeNG]
Currently, you can only set up the application to open a tooltip by specifying 'hover' OR 'focus'.  It seems like the tooltip should also allow you to have the option of 'both', so that a web application can support BOTH users who use their mouse and those that use their keyboard.

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the feedback, we'll wait for community upvotes to implement this feature.

**Comment by zebsola** [comment]
This absolutely should be a possibility for accessibility requirements
